app/functions.py

```diff
@@ -0,0 +1,32 @@
+from app.customer import Customer
+from app.shop import Shop
+from app.car import Car
+
+
+def create_list_of_customers_objects(customers_dict: dict) -> list[Customer]:
+    return [
+        Customer(
+            customer.get("name"),
+            customer.get("product_cart"),
+            customer.get("location"),
+            customer.get("money"),
+            create_car_object_from_dict(customer.get("car")),

```

Comment:
consider using keyword arguments when creating objects to enhance code readability